meson-0.62 is eager to check paramter names. babl's meson.build uses the misspelled "Desription" instead of "description".

--- babl-0.1.90/meson.build.orig	2022-02-21 22:30:17.000000000 +0000
+++ babl-0.1.90/meson.build	2022-04-09 11:02:37.071172311 +0000
@@ -55,7 +55,7 @@
 stability_version_number = (major_version != 0 ? minor_version : micro_version)
 stable = (stability_version_number % 2 == 0)
 
-conf.set10('BABL_UNSTABLE', not stable, Description:
+conf.set10('BABL_UNSTABLE', not stable, description:
   'Define to 1 if this is an unstable version of BABL.')
 
 conf.set       ('BABL_MAJOR_VERSION',    '@0@'.format(major_version))
